### ISSUE TITLE
Standard keyboard navigation shortcuts should be available on macOS

### ISSUE THREAD

**Issue by szhorvat** [ONLYOFFICE/DocumentServer#873]
**Do you want to request a *feature* or report a *bug*?**

Report a bug in keyboard navigation in the word processor on macOS.

**What is the current behavior?**

Position the cursor in a multi-line paragraph. Press Option-RightArrow. The cursor moves by one character. Press Command-RightArrow. The cursor moves by one word.

**What is the expected behavior?**

The expected behaviour on macOS is:

 - Option-Left/RightArrow moves by one word
 - Command-Left/RightArrow moves to the beginning/end of the current line (not the current paragraphs, but the current line, as visible on the screen)

If holding down shift, it should behave as above while extending a selection.

This is how keyboard navigation works in all other macOS apps, including in standard text boxes in the browser. Having to mentally re-adjust when using OnlyOffice is extremely irritating.

**Did this work in previous versions of DocumentServer?**

I do not know (new user).

**DocumentServer version:**

5.4.2

**Operating System:**

macOS 10.14

**Browser version:**

Reproducible with:

Chrome Version 81.0.4044.138 
Firefox version 77.0b4 (beta)

**Comment by szhorvat** [comment]
I downloaded the desktop app, version 5.5.1 (latest as of writing). The behaviour is the same.

**Comment by Rita-Bubnova** [comment]
Hello, @szhorvat.
Thank you. I can confirm - this is know bug, issue 36107 in our internal issue tracker.

**Comment by aol-nnov** [comment]
Hey, @Rita-Bubnova how is it going with issue 36107 in your internal issue tracker? :)

It's a very annoying bug... :(

**Comment by Rita-Bubnova** [comment]
@aol-nnov, Sorry, currently there is no news.

**Comment by aol-nnov** [comment]
sad to hear that...

**Comment by Rita-Bubnova** [comment]
Sorry, we are QA, not developers, but if some issue is opened for long time - our developers couldn't easy implement feature or fix this bug without damaging other part of our app, so it's the reason why issue is not fixed yet.
